metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -44,6 +44,28 @@ second_service="testswarm2"
 port_first_service="8080:80"
 # Port where the second service will run
 port_second_service="8081:80"
+# Mount command for iperf3 (cc-runtime)
+mount="mount -t ramfs -o size=20M ramfs /tmp"
+
+# This function will retrieve the runtime from
+# host A and host B
+function get_runtime {
```

Comment:
I was going to suggest that we don't need this and we use the 'common' set $RUNTIME, but I see this is somewhat specialised as we check both the local and remote machines - so, np :-)